.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
the README mentions this file for example reference of how to run the test in a debugger during developent, for anyone developing on a test may find this a useful reference, not necessarily using it directly in vscode IDE.